autoPyTorch/api/base_task.py

```diff
@@ -781,6 +793,9 @@ def _search(
             raise ValueError("Incompatible dataset entered for current task,"
                              "expected dataset to have task type :{} got "
                              ":{}".format(self.task_type, dataset.task_type))
+        if portfolio_selection not in ["none", "greedy"]:
```

Comment:
yeah this also makes sense as it will allow us to distinguish between the greedy portfolio of the paper and something the user wants to use. and if its None then no portfolio selection. I'll adjust this and rerequest